validator/java/src/test/java/dev/amp/validator/parser/AMPHtmlParserTest.java

```diff
@@ -1056,21 +1056,21 @@ public void testDuplicateDimension() {
 //        }
   }
 
-  @Test
-  public void testCdataViolatesBlacklist() {
-    try {
-      String inputHtml =
-        readFile(
-          "test-cases/css/testCdataViolatesBlacklist.html");
-      final int maxNode = 10000;
-      ValidatorProtos.ValidationResult result =
-        ampHtmlParser.parse(inputHtml, ValidatorProtos.HtmlFormat.Code.AMP4EMAIL, ExitCondition.FULL_PARSING, maxNode);
-      Assert.assertEquals(result.getErrorsCount(), 1, "Expecting to have 1 error");
-      Assert.assertTrue(result.getErrors(0).getCode() == ValidatorProtos.ValidationError.Code.CDATA_VIOLATES_BLACKLIST);
-    } catch (final IOException ex) {
-      ex.printStackTrace();
-    }
-  }
+//  @Test
+//  public void testCdataViolatesBlacklist() {
+//    try {
+//      String inputHtml =
+//        readFile(
+//          "test-cases/css/testCdataViolatesBlacklist.html");
+//      final int maxNode = 10000;
+//      ValidatorProtos.ValidationResult result =
+//        ampHtmlParser.parse(inputHtml, ValidatorProtos.HtmlFormat.Code.AMP4EMAIL, ExitCondition.FULL_PARSING, maxNode);
+//      Assert.assertEquals(result.getErrorsCount(), 1, "Expecting to have 1 error");
+//      Assert.assertTrue(result.getErrors(0).getCode() == ValidatorProtos.ValidationError.Code.CDATA_VIOLATES_BLACKLIST);
+//    } catch (final IOException ex) {
+//      ex.printStackTrace();
+//    }
+//  }
```

Comment:
Can we delete this instead of commenting it out?

Reply:
yeah I can do that 👍making a pr that doesn't include Ryan's changes.